index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
+
 	<div class="req" id="loc_names_famname_reqd">
 	<p class="advisement">Don't require that people supply a family name.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#splitting">Strategies for splitting up names</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p> In cultures such as parts of Southern India, Malaysia and Indonesia, a large number of people have names that consist of a given name only, with no patronym. If you require family names, you may create significant problems in these cultures, as users enter garbage data like &quot;.&quot; or &quot;Mr.&quot; in the family name field just to escape the form.</p>
+</section>
+
+
 
 
 
+
+<section id="loc_names_allowable">
+<h4>Allowable characters</h4>
+
 	<div class="req" id="loc_names_punctuation">
-	<p class="advisement">​ Don't forget to allow people to use punctuation such as hyphens, apostrophes, etc. in names.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<p class="advisement">Allow people to use punctuation such as hyphens, apostrophes, etc. in names, and take into account possible alternative code points for those characters.</p>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>This ensures that names are correctly handled for people such as Dina Asher-Smith and Christopher O'Connell.  Note that the apostrophe may appear as <span class="codepoint" translate="no"><bdi lang="en">&#x0027;</bdi> [<span class="uname">U+0027 APOSTROPHE</span>]</span> or as <span class="codepoint" translate="no"><bdi lang="en">&#x02BC;</bdi> [<span class="uname">U+02BC MODIFIER LETTER APOSTROPHE</span>]</span>, or perhaps even <span class="codepoint" translate="no"><bdi lang="en">&#x2019;</bdi> [<span class="uname">U+2019 RIGHT SINGLE QUOTATION MARK</span>]</span>.  A hyphen may be represented using <span class="codepoint" translate="no"><bdi lang="en">&#x002D;</bdi> [<span class="uname">U+002D HYPHEN-MINUS</span>]</span> or <span class="codepoint" translate="no"><bdi lang="en">&#x2010;</bdi> [<span class="uname">U+2010 HYPHEN</span>]</span>.</p>
```

Comment:
Double hyphens might also be used (in Japanese).

For foreign names that include hyphens, a double hyphen may be used instead of a hyphen. For example, if someone's family name is Zeta-Jones, when she fills in a Japanese form, her family name may appear as: `ゼタ゠ジョーンズ`.

Reply:
Added.